tasks/79_calculate_primoral_of_a_number/calculate_primoral_of_a_number.js

```diff
@@ -0,0 +1,18 @@
+export const numPrimorial = (num) => {
+  let primorial = 1;
+  let count = 0;
+  for (let j = 2; count < num; j++) {
+    let prime = true;
+    for (let i = 2; i <= parseInt(Math.sqrt(j)) && prime; i++) {
```

Comment:
looks like a code which can be replaced to separate function which you will call in if with current j